Source/MLXNN/Module.swift

```diff
@@ -392,7 +392,7 @@ open class Module {
     ///
     /// This passes `verify: .none`.  Note that there may still be `fatalErrors()` if
     /// for example an `MLXArray` is set on a `Module`.
-    public func update(parameters: ModuleParameters) {
+    public func update(parameters: ModuleParameters) -> Self {
```

Comment:
The python functions allow chaining.  That isn't as typical in swift (though very much it was in old ObjC) but we can give it a try!

Reply:
Sounds good!